### PR TITLE
Refactor buttons

### PR DESCRIPTION
**Pull request by GusBaamonde**
Refactor buttons

### CODE REVIEW

**Comment by andresmoschini** [comment]
> Refactor buttons

Could you add a better PR description?

For example, what did you do with the buttons and why.

**Comment by andresmoschini** [comment]
image

**Comment by IgnacioRodrigues** [comment]
Good work Gus, maybe add link cdn with the build number

**Comment by GusBaamonde** [comment]
> Refactor buttons

> od work Gus, maybe add link cdn with the build number

https://cdn.fromdoppler.com/doppler-ui-library/build72/index.html

File: assets/scss/templates/_guidelines.scss [hunk 1]
```diff
@@ -141,51 +142,75 @@
...
+  position: absolute;
+  transition: all 0.3s;
+  top: -15px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.description-button {
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_guidelines.scss]: maybe the name is button-description . What do you think?

File: assets/scss/templates/_guidelines.scss [hunk 2]
```diff
@@ -141,51 +142,75 @@
...
+  position: relative;
+  min-height: 260px;
+  transition: all 0.3s;
+}
+
+.box-button::before,
+.btn::after {
+  content: '';
+  position: absolute;
+  transition: all 0.3s;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_guidelines.scss]: it's not de default transition?

File: assets/scss/templates/_guidelines.scss [hunk 3]
```diff
@@ -141,51 +142,75 @@
...
+  &:hover:after {
+    transform: scale(0, 0);
+    opacity: 0;
+  }
+}
+
+.box-button {
+  position: relative;
+  min-height: 260px;
+  transition: all 0.3s;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_guidelines.scss]: it's not the default transition?

File: assets/scss/templates/_guidelines.scss [hunk 4]
```diff
@@ -141,51 +142,75 @@
...
+    transform: scale(1.3, 1.3);
+  }
+
+  &:hover:before {
+    opacity: 1;
+    transform: scale(1, 1);
+  }
+
+  &:after {
+    transition: all 0.3s;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_guidelines.scss]: it's not the default transition?

File: assets/scss/templates/_guidelines.scss [hunk 5]
```diff
@@ -141,51 +142,75 @@
...
-  width: 300px;
+  width: 25%;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
   text-align: left;
+
+  &:before {
+    opacity: 0;
+    border-radius: 3px;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_guidelines.scss]: it's not the default radius?

File: assets/scss/templates/_guidelines.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/templates/_guidelines.scss]: Tenes razon Fede ya lo cambio!

File: assets/scss/templates/_guidelines.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/templates/_guidelines.scss]: Nop, create now! ;)

File: assets/scss/templates/_guidelines.scss (hunk 3 above)
Comment (GusBaamonde) [reply assets/scss/templates/_guidelines.scss]: Nop, create now!

File: assets/scss/templates/_guidelines.scss (hunk 4 above)
Comment (GusBaamonde) [reply assets/scss/templates/_guidelines.scss]: Nop, create now!

File: assets/scss/templates/_guidelines.scss (hunk 5 above)
Comment (GusBaamonde) [reply assets/scss/templates/_guidelines.scss]: Nop, create now! thanks @fcoronelmakingsense !

**Pull request merged**